compiler/daml-extension/BUILD.bazel

```diff
@@ -91,19 +92,18 @@ genrule(
         export HOME=/does-not-exist
         set -euo pipefail
         TMP_DIR=$$(mktemp -d)
-        cleanup () { rm -rf $$TMP_DIR || return; }
+        cleanup () {{ rm -rf $$TMP_DIR || return; }}
```

Comment:
Yes: because we add a call to .format at the end of that string, Bazel will take anything in squiggly brackets as a substitution; the double brackets prevent that.